src/pages/home.js

```diff
@@ -1,5 +1,24 @@
 import React from "react";
+import "./home.css";
+import logo from "./img/logo-img.png";
+import { Link } from 'react-router-dom';
 
 export default function Home() {
-  return <h2>Home Page Test</h2>;
+  return (
+    <>
+      <section className="hero is-link is-fullheight-with-navbar">
+        <div className="hero-body body">
+          <div className="container">
+            <img src={logo} alt="Logo" class="logo-img" />
+            <h1 className="title" id="title">InvisionAR</h1>
+            <p className="slogan">InVisioning the future...</p>
+            <Link to="/about"><button class="button is-primary" id="start-btn" to="#"><strong>GET STARTED</strong></button></Link>
+            <div className="footer">
```

Comment:
I don't think a fixed footer is needed for the landing page. You could use [hero-foot](https://bulma.io/documentation/layout/hero/#fullheight-hero-in-3-parts) instead to add a footer without scrolling.